packages/zipkin/index.d.ts

```diff
@@ -150,8 +150,6 @@ declare namespace zipkin {
     record: (rec: any) => void;
   }
 
-  const serializeSpan: () => void;
```

Comment:
ps a later request can formalize the typescript of accessing encoders.. we shouldn't do it, yet as for example the result isn't standard. Specifically, json returns a string whereas thrift returns a Buffer. That's the long story why I didn't replace this with anything.. we'll revisit later and do it more formally